### DISCUSSION TITLE
How alive is beets?

### DISCUSSION THREAD

**Discussion by etdr** [beetbox/beets#4920] [General]
I had planned to beets my music library all the way through college and never did so, and now I'm getting back to this project. I thought I had read at one point that the development on this project was at least stalled, and the last official release was indeed like two years ago, but I also see the newest commit to the project was today. I'm wondering:
1. What are the long term development/support prospects for this project?
2. Are there any caveats I watch out for with using beets in 2023?

Have always been fascinated by this program and am looking to finally use it, just wanted to see if I could get some clarity on this first.

**Comment by Serene-Arc** [comment]
Releases have been...less than regular, let's just say that. You'd have to ask @sampsyo for details on the next one. But beets is still very much alive!

There's tons of active development going on and the code base is being expanded and fixed constantly. I joined the dev group relatively recently and there's lots of things in the works. Right now, some members of our team are working towards merging a large update to the UI of beets, which is exciting! I'm working on a few things to tidy up the code and make development and testing easier, and other devs have their own areas they're working on when they have the chance. This is an open-source project, so most devs don't spend hours a day on it but we try to keep things moving (even if the speed isn't exactly lightning-fast).

You'll also notice that the community here is quite lively. Try subscribing to the repository and you'll see that there's always lots of new bugs and discussion on PRs and issues. There's lots of people still using the program and a pretty wide assortment of related plugin projects.

If you're going to start using beets and want the latest and greatest, then I'd recommend installing from the development branch. As you've spotted, it's two years ahead of the last release and there's a lot of great features and bug fixes in there. The docs are very comprehensive; just make sure you're on the latest version and you'll do fine. Don't be scared by the development branch either; all PRs are tested and used prior to merging so it's not likely to crash or cause issues. If it does, open a bug! But for what it's worth, personally, I use it for both music and audiobooks and 99.9% of the time it works 100% of the time :) It really is a very stable branch and you can pin it to the latest commit and update as needed.

Welcome to beets! I hope you use it, it really is a great tool.

**Reply by etdr** [reply]
Thank you for the thorough response! I will definitely try the development branch and submit issues if I run into any.

I honestly didn't even know there *was* a GUI, I will try that too (although I am not afraid of using the CLI).

**Reply by Serene-Arc** [reply]
Oh, sorry, there isn't a GUI. We've just merged/are finishing up a big update to the CLI interface.

**Comment by etdr** [comment]
Before I open a new thread for questions, I just want to confirm where the best place to ask quick questions is so I'm not cluttering up Discussions with too many threads.. if you don't have a chat room or Discord, is Github Discussions optimal? I can also try searching on the old message board, if that content is still relevant.

**Reply by Serene-Arc** [reply]
Yes, this is the right place :)
